### DISCUSSION TITLE
Real time kernel patch

### DISCUSSION THREAD

**Discussion by noob502** [OE4T/meta-tegra#1736] [General]
Hello, 
is there any real time kernel patch for mickledore branch ? 
I have tried patch from https://cdn.kernel.org/pub/linux/kernel/projects/rt/5.10/, but it is not working.  

Is the mickledore branch already provide the patch ?

Thanks !

**Comment by madisongh** [comment]
That branch is based off L4T R35.4.1, and [according to the NVIDIA documentation](https://docs.nvidia.com/jetson/archives/r35.4.1/DeveloperGuide/text/SD/Kernel/KernelCustomization.html#to-build-the-real-time-kernel) it includes the PREEMPT_RT patches.

You should be able to adapt the approach on this rather old [wiki page](https://github.com/OE4T/meta-tegra/wiki/Applying-PREEMPT-RT-Patches-dunfell-l4t-r32.4.3).

**Comment by noob502** [comment]
Current branch is on linux-tegra-5.10 and in the wiki , it suggested to fork linux-tegra-4.9 which has patch based on r32.4. Are you suggested to apply  the r32.4 patche or only follow the procedure.

**Reply by madisongh** [reply]
Just the procedure - fork the repo, run the script included in the source tree to apply the patches, check in the patched sources to a new branch, add a bbappend for the kernel recipe to point to your new branch.
